Task_2/src/model/Paragraph.java

```diff
@@ -0,0 +1,46 @@
+package model;
+
+import java.util.ArrayList;
+
+
+public class Paragraph implements Component<Paragraph> {
+
+    private final static String DELIMITER_FOR_SENTENCE = "\\. ";
+
+    private ArrayList<Component> sentences;
+
+    public Paragraph(String s) {
+        this.sentences = readText(s, DELIMITER_FOR_SENTENCE);
+        Counter.sentence = Counter.sentence + sentences.size();
+    }
+
+
+    @Override
+    public ArrayList<Component> readText(String text, String delimiter) {

```

Comment:
Try to reuse this code and not duplicate it in other parts

Reply:
Ok